pypesto/visualize/parameters.py

```diff
@@ -23,7 +23,8 @@ def parameters(
         colors: Optional[Union[List[float], List[List[float]]]] = None,
         legends: Optional[Union[str, List[str]]] = None,
         balance_alpha: bool = True,
-        start_indices: Optional[Union[int, Iterable[int]]] = None
+        start_indices: Optional[Union[int, Iterable[int]]] = None,
+        scale: Optional[Tuple[float, float]] = None,
```

Comment:
"scale" is a bit over-used. "(re)scaled_bounds"?

Reply:
replaced